package.json

```diff
@@ -28,5 +28,16 @@
     "cross-env": "^3.2.3",
     "lerna": "^2.0.0-beta.38-31-g0749ea6",
     "nyc": "^10.2.0-candidate.0"
+  },
+  "nyc": {
+    "exclude": [
+      "**/test/**",
+      "**/dist/**",
+      "**/bin/**"
```

Comment:
TBPH I'm not a huge fan of this because it seems brittle (i.e. what if we hypothetically wanted to cover something in a `dist/` folder?) but I think the compromise works because of the use case. Let me know what you all think :)